organization_integration_test.go

```diff
@@ -272,6 +304,29 @@ func TestOrganizationsUpdate(t *testing.T) {
 		assert.Equal(t, orgTest.Name, org.Name)
 		assert.Equal(t, orgTest.Email, org.Email)
 	})
+
+	t.Run("with default execution mode of 'agent'", func(t *testing.T) {
+		// this helper creates an organization and then updates it to use a default agent pool
+		org, orgAgentTestCleanup := createOrganizationWithDefaultAgentPool(t, client)
+		assert.Equal(t, "agent", org.DefaultExecutionMode)
+		assert.NotNil(t, org.DefaultAgentPool)
+		t.Cleanup(orgAgentTestCleanup)
+	})
+
+	t.Run("with default execution mode of 'remote'", func(t *testing.T) {
```

Comment:
this 2 test cases seem to be doing the same thing, i.e test case
`t.Run("with default execution mode of 'agent'", func(t *testing.T) {...`
and
`t.Run("with default execution mode of 'remote'", func(t...`.

Should they be merged as one?